qa/rpc-tests/rawtransactions.py

```diff
@@ -56,6 +57,14 @@ def run_test(self):
         rawtx   = self.nodes[2].createrawtransaction(inputs, outputs)
         rawtx   = self.nodes[2].signrawtransaction(rawtx)
 
+        # check missing input also using verifyrawtransactions
+        err   = self.nodes[2].verifyrawtransactions([rawtx['hex']])
+        assert(err is not None)
+        assert(err['index'] == 0)
+        assert(err['code'] == 16)
```

Comment:
Hmm, interesting, so CheckSequenceLocks actually looks up the inputs, moving it after HaveInputs